README.md

```diff
@@ -129,6 +129,12 @@ This application uses both a PostgreSQL database as its main storage as well as
 
 For more information, see [*Database*](docs/database.md).
 
+## SSL Certificates Renewal
+
+SSL Certificates for the Dev and Test environments are automatically renewed using let's encrypt through a tool called openshift-acme.
```

Comment:
Improper capitalization.
* "Certificates" should not be capitalized.
* Let's Encrypt is a proper noun and should be capitalized as such.
* "openshift-acme" should potentially be wrapped in backticks to make it clearer as a package name. Less important, though.